tutorials/VI_in_BM.ipynb

```diff
@@ -0,0 +1,2009 @@
+{
```

Comment:
<p>Why is this the true posterior?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/facebookresearch/beanmachine/pull/1633/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

Reply:
It's the closed form for the Gaussian unknown mean model where there is a single observation.

Reply:
I see. Can we add a text cell or comment that this is due to Gaussian location parameter conjugacy?

Reply:
Yep!